l5kit/l5kit/tests/rasterization/rasterizer_e2e_test.py

```diff
@@ -59,6 +59,20 @@ def setup_rasterizer_artifacts_and_config(temp_folder: str, cfg: dict) -> None:
     with open(os.path.join(temp_folder, sat_im_metadata_filename), "w") as ft:
         json.dump(sat_im_transform_json_content, ft)
 
+    dataset_meta_key = "dataset_meta.json"
```

Comment:
We have two meta data json file? One is `meta.json` defined in yaml another is `dataset_meta.json` used in test?

Reply:
Let me change the name for this one. Anyway, tests are going to be completely refactored next week :)